include/openssl/evp.h

```diff
@@ -1142,5 +1142,6 @@ BSSL_NAMESPACE_END
 #define EVP_R_INVALID_PEER_KEY 134
 #define EVP_R_NOT_XOF_OR_INVALID_LENGTH 135
 #define EVP_R_EMPTY_PSK 136
+#define EVP_R_INVALID_PSS_MD 137
```

Comment:
Good call-out, but I'd suggest starting with 1000 instead of 200, making sure that Boring's never gonna override our errors and it's easier for us to track know what've added.

Reply:
Addressed.